### ISSUE TITLE
Conduct stakeholder interviews 

### ISSUE THREAD

**Issue by dianagriffin** [department-of-veterans-affairs/abd-vro#1454]
**User story**
As a new VRO designer, I want to understand our VA partners' perspectives on the goals of our work and what success looks like for our team and our product, so that I can shape my future work to deliver on those goals.

**Acceptance Criteria**

1. Stakeholder interviews conducted with Premal Shah, Emily Theis, Zach Goldfine, Steve Albers, Shannon Ford

**Not included in this work**

This ticket doesn't include synthesis/next steps; we'll spin up a new ticket to cover next steps from these interviews

**Comment by dianagriffin** [comment]
@megameg I added Steve (OCTO Engineering lead) and Shannon (OCTO Design lead) to the list of stakeholders; I think both will have great perspective!

**Comment by megameg** [comment]
Schedule with:

- [ ] Premal Shah
- [ ] Emily Theis
- [ ] Zach Goldfine
- [ ] Steve Albers
- [ ] Shannon Ford

Create script

- [ ]  (insert link)

**Comment by megameg** [comment]
[Interview notes are available here ](https://docs.google.com/document/d/1IaLhFtxb6ebAcRPMuBz-e16XfsbUb8A5iQno19rTYAo/edit?usp=sharing)

**Comment by megameg** [comment]
Waiting on Zach's availability -- will have the last interview next Monday.
